hack/run-update-testing.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
do we need this file here ? this change should come before https://github.com/openshift/cluster-node-tuning-operator/pull/517 , so should'nt we add this only in #517  ?

Reply:
This is a separate bash script for running the updating test.